Documentation/op-guide/configuration.md

```diff
@@ -279,6 +279,10 @@ Follow the instructions when using these flags.
 + Enable runtime profiling data via HTTP server. Address is at client URL + "/debug/pprof/"
 + default: false
 
+### --enable-grpc-time-histogram
```

Comment:
how about --metrics = basic, extensive (enable histogram) ?

I am not a fan of too specific flag.

/cc @heyitsanthony